apps/system/js/bluetooth.js

```diff
@@ -193,11 +192,15 @@ var Bluetooth = {
     var req = bluetooth.getDefaultAdapter();
     req.onsuccess = function bt_gotDefaultAdapter(evt) {
       self.defaultAdapter = req.result;
-      self.initWithAdapter(self.defaultAdapter);
+      self._adapterAvailableHandler(self.defaultAdapter);
+      // Dispatch bluetooth adapter available state to system.
+      window.dispatchEvent(new CustomEvent('bluetooth-available', {
```

Comment:
Since we put `getAdapter` in service query, why we still dispatch `bluetooth-available` event?